sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample1_DetectEntireSeriesAnomaly.md

```diff
@@ -0,0 +1,73 @@
+﻿# Detect Entire Series Anomaly
+This sample shows how to detect all the anomalies in the entire time series.
+
+To get started, make sure you have satisfied all the prerequisites and got all the resources required by [README][README].
+
+## Create an AnomalyDetectorClient
+
+To create a new `AnomalyDetectorClient` you need the endpoint and credentials from your resource. In the sample below you'll use an Anomaly Detector API key credential by creating an `AzureKeyCredential` object.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateAnomalyDetectorClient
```

Comment:
CI is failing because you didn't use the snippets anywhere is `.cs` files. You can refer [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeContentFromUri.cs#L25) for the usage of snippets.